assemble/bin/accumulo-cluster

```diff
@@ -32,9 +32,9 @@ Commands:
   stop                       Stops Accumulo cluster
   kill                       Kills Accumulo cluster
   start-non-tservers         Deprecated. Starts all services except tservers
-  start-servers [--all|--tservers|--no-tservers|--sservers [group]|--compactors [group]]
+  start-servers [--all|--tservers [group]|--no-tservers|--sservers [group]|--compactors [group]]
```

Comment:
I'm thinking we should either get rid of the `--no-tservers` option or add `--no-sservers` and `--no-compactors`.

Reply:
I think I'm not familiar with the use case for starting everything but the tservers.

Reply:
That was a carry-over from the `start-non-tservers` behavior. I think it's probably best if we remove the `--no-tservers` option vs adding extra flags.

Reply:
Flags have been removed.